repos/system_upgrade/el7toel8/actors/ls/actor.py

```diff
@@ -0,0 +1,17 @@
+from leapp.actors import Actor
+from leapp.tags import LateTestsPhaseTag, IPUWorkflowTag
+from leapp.libraries.stdlib import run
+
+
+class Ls(Actor):
+    """
+    No documentation has been provided for the ls actor.
+    """
+
+    name = 'ls'
+    consumes = ()
+    produces = ()
+    tags = (LateTestsPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        run(['ls', '-lR', '/var/log'])
```

Comment:
I guess this actor is not supposed to be part of the PR

Reply:
Oh shit, my bad. This slipped in. :D